build-system/tasks/storybook/amp-env/decorator.js

```diff
@@ -24,22 +24,23 @@ export default makeDecorator({
   wrapper: (getStory, context) => {
     const contents = render(getStory(context));
 
+    // DO NOT SUBMIT: QQQ: local-vs-CDN script URLs.
     const ampHtml = `
         <!doctype html>
         <html amp lang="en">
         <head>
             <meta charSet="utf-8" />
             <title>AMP Page Example</title>
             <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
-            <script async src="https://cdn.ampproject.org/v0.js"></script>
+            <script async src="http://localhost:8000/dist/amp.js"></script>
```

Comment:
I was hoping people would be using the [Chrome extension](https://github.com/ampproject/amphtml/tree/master/testing/local-amp-chrome-extension) for now to test on local, until we implement a panel inside storybook which would allow to switch between amp builds, bento mode and module/no-module (I am having some trouble with integrating the panel but a PR should be coming).

Reply:
Hmm. Interesting. Let's try that. I'll close this now.